.gitignore

```diff
@@ -1,10 +1,19 @@
-deployment
-/releases
-/.settings
-*.launch
+# Ignore Gradle project-specific cache directory
+.gradle
+
+# Ignore Gradle build output directory
+build
+
+# Ignore Mac and vscode files
+.DS_Store
+.vscode
+
+# Ignore files generated by Studio Pro
 .classpath
 .project
-proxies
-*/javasource/system/actions/VerifyPassword.java
-*/javasource/system/UserActionsRegistrar.java
-src/AuditTrailModule/javasource/system/
\ No newline at end of file
+**/deployment/
+.gradle
+.settings

```

Comment:
Can we please verify if this is generate because I can't see it in anywhere?

Reply:
Some of the files I added are created either by Eclipse or vscode. I see no harm in keeping them here just to make using the repo easier.

Reply:
Okay then.